### PR TITLE
chore(core): bump web-console version to 0.0.5

### PR DESCRIPTION
**Pull request by argshook**
### Added

- Add popup to show keyboard shortcuts [a3a10c9b](https://github.com/questdb/ui/commit/a3a10c9b8678de761a1e9ce1ab380837d9121e2a)

### Added

- Add info about an available update to QuestDB

[changelog](https://github.com/questdb/ui/blob/main/packages/web-console/CHANGELOG.md)

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <javac.target>11</javac.target>
         <argLine>-ea -Dfile.encoding=UTF-8</argLine>
         <test.exclude>None</test.exclude>
-        <web.console.version>0.0.3</web.console.version>
+        <web.console.version>0.0.5</web.console.version>
```
Comment (bluestreak01) [inline core/pom.xml:38]: why did we skip a version?

Comment (argshook) [reply core/pom.xml:38]: because we forgot about 0.0.4

Comment (argshook) [reply core/pom.xml:38]: i mean, 0.0.4 was released, but not added to this repo

Comment (bluestreak01) [reply core/pom.xml:38]: ok, cool

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**